src/models/athlete.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const mongoose = require('mongoose')
+const joigoose = require('joigoose')(mongoose)
+const joi = require('joi')
+const uuid = require('uuid/v1')
```

Comment:
afair mongo uses `v4`

Reply:
I looked it up and you are correct.  I specifically choose v1 for uniqueness vs v4 for randomness since I am not concerned about the possibility of the loss of anonymity. In reality, I think either would be fine.